test/e2e/testsuite/testsuite.go

```diff
@@ -539,6 +549,54 @@ func FileExistsInCRCHome(fileName string) error {
 	return err
 }
 
+func BundleIsCachedOrNot(presetName string, isOrNot string) error {
+
+	bundle := ""
+	if presetName == "podman" {
+		bundle = constants.GetDefaultBundle(preset.Podman)
+	} else {
+		bundle = constants.GetDefaultBundle(preset.OpenShift)
+	}
+
+	theFile := filepath.Join(CRCHome, "cache", bundle)
+
+	_, err := os.Stat(theFile)
+
+	if os.IsNotExist(err) && isOrNot == "is" {
+		return fmt.Errorf("file %s does not exists, error: %v ", theFile, err)
+	}
+
+	if err == nil && isOrNot == "is not" {
+		return fmt.Errorf("file %s exists when it should not exist", theFile)
+	}
+
+	return nil
+}
+
+func RemoveBundleFromCache(presetName string) error {
+
+	bundle := ""
+	if presetName == "podman" {
+		bundle := constants.GetDefaultBundle(preset.Podman)
+		dirName := bundle[:len(bundle)-10] // remove `.crcbundle` suffix
```

Comment:
`dirName = strings.TrimSuffix(bundle, ".crcbundle")`
This can be moved outside of the `if{}` block as you have the same code in botoh branches.

Reply:
`TrimSuffix`, good idea - it looks much less offensive that way! Reorganized that `if`  as you suggested (I think).